Formula/satysfi.rb

```diff
@@ -0,0 +1,29 @@
+class Satysfi < Formula
+  desc "Statically-typed, functional typesetting system"
+  homepage "https://github.com/gfngfn/SATySFi"
+  url "https://github.com/gfngfn/SATySFi/archive/v0.0.2.tar.gz"
+  sha256 "91b98654a99d8d13028d4c7334efa9d8cc792949b9ad1be5ec8b4cbacfaea732"
+
+  depends_on "opam" => :build
+  depends_on :x11 => :build
+
+  def install
+    mktemp do
```

Comment:
Can we use [this sort of format](https://github.com/Homebrew/homebrew-core/blob/71576cecdc8bf61550134e56ce837e6fc27dea4a/Formula/docker-machine-driver-xhyve.rb#L47-L48) here?

Reply:
I don't think so. I added this mktemp block to prevent opam from trying copying a directory into that directory itself. This is not simply a matter of format.